sunpy/util/scraper.py

```diff
@@ -61,8 +68,13 @@ class Scraper:
     The ``now`` attribute does not return an existent file, but just how the
     pattern looks with the actual time.
     """
-    def __init__(self, pattern, **kwargs):
-        self.pattern = pattern.format(**kwargs)
+    def __init__(self, pattern, regex=False, **kwargs):
+        if regex:
+            self.pattern = pattern
+            if kwargs:
+                warnings.warn('regexp being used, the extra arguments passed are being ignored')
```

Comment:
should this also use a `SunpyUserWarning`?

 i.e. 
`warnings.warn('regexp being used, the extra arguments passed are being ignored', SunpyUserWarning )`

Reply:
Thanks!